### DISCUSSION TITLE
借助craco如何实现less模块化

### DISCUSSION THREAD

**Discussion by Wykyoyo** [ant-design/ant-design#29037] [Q&A]
import styles from './index.less';
我打算className={styles.app}，但是失败了，求助我该如何调整以达到效果
![0](https://user-images.githubusercontent.com/32306550/105794106-48d44900-5fc5-11eb-981d-9d6afa5905f3.png)
![1](https://user-images.githubusercontent.com/32306550/105794111-4bcf3980-5fc5-11eb-912f-d6b5d2862414.png)

**Comment by Xiaoming985** [comment]
同求方案
